store/types/pruning.go

```diff
@@ -11,10 +11,12 @@ const (
 )
 
 var (
-	// PruneDefault defines a pruning strategy where the last 100 heights are kept
-	// in addition to every 100th and where to-be pruned heights are pruned at
-	// every 10th height.
-	PruneDefault = NewPruningOptions(100, 100, 10)
+	// PruneDefault defines a pruning strategy where the last 9072000 heights are
+	// kept in addition to every 100th and where to-be pruned heights are pruned
+	// at every 10th height. The last 9072000 heights are kept assuming the typical
+	// block time is 5s and typical unbonding period is 21 days. If these values
```

Comment:
Assuming block time is 5s, there's 3600*24/5 = 17280 blocks per day, so 362880 in 21 days.

Reply:
Ahh good catch!